src/page/shipping-step.page.ts

```diff
@@ -5,7 +5,7 @@ export class ShippingStepPage {
   private acceptTerms: ElementFinder;
 
   constructor () {
-    this.shippingStep = $('#form > p > button > span');
+    this.shippingStep = $('.cart_navigation span');
```

Comment:
Try to find an attribute for this button that makes it unique for the page, and use it as a locator.

Reply:
Done!!